### PR TITLE
glyph: Comment the actual char for easy visualization

### PR DESCRIPTION
**Pull request by bew**
It helps visualize the char of the glyph when reading the code :smiley:

### CODE REVIEW

File: wezterm-gui/src/glyphcache.rs [hunk 1]
```diff
@@ -361,7 +361,7 @@ impl BlockKey {
         let c = c as u32;
         Some(match c {
             // BOX DRAWINGS LIGHT HORIZONTAL
-            0x2500 => Self::Poly(&[Poly {
+            0x2500 => Self::Poly(&[Poly { // ─
```
Comment (wez) [inline wezterm-gui/src/glyphcache.rs:364]: I'd prefer it if the comment was on the line above the codepoint value; for the items that are labelled with the name, let's put it in with the comment; either at the LHS or the RHS of the line (RHS show in this suggestion, but I wonder if LHS might be better)

```suggestion
            // BOX DRAWINGS LIGHT HORIZONTAL  ─
            0x2500 => Self::Poly(&[Poly {
```

File: wezterm-gui/src/glyphcache.rs [hunk 2]
```diff
@@ -3277,117 +3277,117 @@ impl BlockKey {
...
-            0x2592 => Self::Full(BlockAlpha::Medium),
-            0x2593 => Self::Full(BlockAlpha::Dark),
-            0x2594 => Self::Upper(1),
-            0x2595 => Self::Right(1),
-            0x2596 => Self::Quadrants(Quadrant::LOWER_LEFT),
-            0x2597 => Self::Quadrants(Quadrant::LOWER_RIGHT),
-            0x2598 => Self::Quadrants(Quadrant::UPPER_LEFT),
-            0x2599 => {
+            0x2590 => Self::Right(4), // ▐
+            0x2591 => Self::Full(BlockAlpha::Light), // ░
```
Comment (wez) [inline wezterm-gui/src/glyphcache.rs:3288]:
```suggestion
            // ░
            0x2591 => Self::Full(BlockAlpha::Light),
```

**Review by wez — CHANGES_REQUESTED** [review]
Thanks for this; now that we have these glyphs in wezterm I'm all for being able to see them in the code!

The `cargo fmt` check is failing because the commenting positioning isn't how it likes it; I think it would be good to put the comments before the codepoint -- see suggestions!

File: wezterm-gui/src/glyphcache.rs (hunk 1 above)
Comment (bew) [reply wezterm-gui/src/glyphcache.rs:364]: Hmm was thinking about this, but it wasn't easy to automate in vim, so I made this first impl for now.
I think LHS might be better, also I'd like to surround the char with `(X)` or `[X]` to better show where the char begins/ends (useful for the partial blocks)
What do you think?

File: wezterm-gui/src/glyphcache.rs (hunk 2 above)
Comment (bew) [reply wezterm-gui/src/glyphcache.rs:3288]: So I add a comment line above ALL the oneline glyphs overrides?

Comment (wez) [reply wezterm-gui/src/glyphcache.rs:3288]: I think the comment works better when it precedes the codepoint value, as it is in the same horizontal position, which makes it easier to visually scan.

In hindsight, I should have put the codepoint description there for all of those glyphs when I started adding them; it didn't feel necessary when I started, but it definitely made sense when I did the box drawing glyphs.

If you're up for automating adding those in, that would be fantastic, but it's not required in this PR!

File: wezterm-gui/src/glyphcache.rs (hunk 1 above)
Comment (wez) [reply wezterm-gui/src/glyphcache.rs:364]: `[X]` sounds good, and I agree: LHS feels easier to visually scan

File: wezterm-gui/src/glyphcache.rs (hunk 2 above)
Comment (bew) [reply wezterm-gui/src/glyphcache.rs:3288]: I can automate it quite easily if I have a list of all the codepoint descriptions and their corresponding 0x1234 notation, find me that and I'll do it in this PR 🙃

**Comment by bew** [comment]
Okay, everything should be good!

Here is a little relaxing screencast of navigating in the code....
(me? crazy? nooo)

https://user-images.githubusercontent.com/9730330/124365675-cde51d80-dc49-11eb-805c-a9903aa742b2.mp4

**Comment by wez** [comment]
This is great, many thanks!

**Pull request merged**